### PR TITLE
ci: Use more common 4-digit year format for dep id

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
I consider "Dependency cron 200906" hard to read until one understands that "20" are the last two digits of a year.

**Comment by coolo** [comment]
That's one of the smaller 2020 problems :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -131,9 +131,9 @@ jobs:
       - run:
           name: "Prepare dependency PR"
           command: |
-            depid=$(date +%y%m%d)
+            depid=$(date +%Y%m%d)
```
Comment (Martchus) [inline .circleci/config.yml:134]: I'd even prefer some dashes.

Comment (okurz) [reply .circleci/config.yml]: I originally tried ISO8601 which would be corresponding to `%Y-%m-%d` (although the format without hyphen is also allowed in the standard) but the timestamp including the time would include the timezone with a `+` sign and `:` which is forbidden in git branch name.

Ok, so I changed the two statements to use ISO8601 with the abbreviated time format when the time is included but without the timezone. Example: "2020-09-08T141337".

**Review by perlpunk — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Pull request merged**